### ISSUE TITLE
Problem with Rasa 2.0 versions

### ISSUE THREAD

**Issue by malamasn** [koenvervloesem/rasa-docker-arm#2]
I tried building an image for the latest rasa version (2.0.5 today) and came up with this error:

![image](https://user-images.githubusercontent.com/40325817/99063592-62ace500-25ad-11eb-96d2-a8726ca7b7ba.png)

Any help, please? Is it because of the armv7 architecture?

**Comment by koenvervloesem** [comment]
I don't think it has anything to do with the architecture (supposing you're building this on armv7). I haven't tried building a Rasa 2.0 image yet because a lot of the TensorFlow dependencies have made major version bumps there and I haven't found the time yet to try to build or integrate these. I suspect you will encounter a lot of these issues that need to be solved one by one before a Rasa 2.0 Docker image is possible.
